app/src/main/java/org/mozilla/focus/browser/BrowserFragment.kt

```diff
@@ -165,20 +171,54 @@ class BrowserFragment : IWebViewLifecycleFragment() {
                 // so we need to do our own caching: see FocusedDOMElementCache for details.
                 if (!isVisible) { webView?.focusedDOMElement?.cache() }
             }
+            registerForContextMenu(browserOverlay.tileContainer)
         }
 
         layout.progressBar.initialize(this)
 
         return layout
     }
 
+    override fun onContextItemSelected(item: MenuItem): Boolean {
```

Comment:
nit: you can save an indent with:
```kotlin
override fun onContextItemSelected(item: MenuItem): Boolean = when (item.itemId) {
    R.id.remove -> ...
```

The return type isn't strictly necessary but it helps to be explicit (e.g. `when` might infer a `Any?` return type depending on the values you return) – I'm still going back and forth whether I prefer to declare the types or not

Reply:
We discussed that we should keep it this way since the inner nested stuff gets a bit complicated and we need to return early in some cases.